src/components/CodeEditor/index.jsx

```diff
@@ -16,6 +16,9 @@ import './json-lint';
 @withStyles({
   root: {
     width: '100%',
+    '& .CodeMirror-placeholder': {
+      color: '#656469 !important',
+    },
```

Comment:
```diff
   root: {
     width: '100%',
     '& .CodeMirror-placeholder': {
-      color: '#656469 !important',
+      // Same as demo
+      // https://github.com/codemirror/CodeMirror/blob/e3b9957f3e46aad5c0c2fd309f65b6fec10d267d/demo/placeholder.html#L14
+      color: '#999',
     },
   },
 })
```

Was there a reason for adding `!important`?

Reply:
I tried without `!important`, but the color didn't change.